src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelTrace.Connections.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure;
+
+internal sealed partial class KestrelTrace : ILogger
+{
+    public void ConnectionAccepted(string connectionId)
+    {
+        ConnectionsLog.ConnectionAccepted(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionStart(string connectionId)
+    {
+        ConnectionsLog.ConnectionStart(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionStop(string connectionId)
+    {
+        ConnectionsLog.ConnectionStop(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionPause(string connectionId)
+    {
+        ConnectionsLog.ConnectionPause(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionResume(string connectionId)
+    {
+        ConnectionsLog.ConnectionResume(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionKeepAlive(string connectionId)
+    {
+        ConnectionsLog.ConnectionKeepAlive(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionRejected(string connectionId)
+    {
+        ConnectionsLog.ConnectionRejected(_connectionsLogger, connectionId);
+    }
+
+    public void ConnectionDisconnect(string connectionId)
+    {
+        ConnectionsLog.ConnectionDisconnect(_connectionsLogger, connectionId);
+    }
+
+    public void NotAllConnectionsClosedGracefully()
+    {
+        ConnectionsLog.NotAllConnectionsClosedGracefully(_connectionsLogger);
+    }
+
+    public void NotAllConnectionsAborted()
+    {
+        ConnectionsLog.NotAllConnectionsAborted(_connectionsLogger);
+    }
+
+    public void ApplicationAbortedConnection(string connectionId, string traceIdentifier)
+    {
+        ConnectionsLog.ApplicationAbortedConnection(_connectionsLogger, connectionId, traceIdentifier);
+    }
+
+    public static partial class ConnectionsLog
+    {
+        [LoggerMessage(39, LogLevel.Debug, @"Connection id ""{ConnectionId}"" accepted.", EventName = "ConnectionAccepted")]
```

Comment:
Could you order the log message methods by event id? e.g. 1, 2, 3, 4.

These were reordered at one point so they are grouped together by logic, but we don't really care. The most useful bit of info is what is the last ID. If they're ordered we can simply look at the bottom of the class.

Reply:
Yup, re-ordered in both files.